services/common/src/main/java/com/linkedin/openhouse/common/stats/model/CommitEventPartitions.java

```diff
@@ -0,0 +1,50 @@
+package com.linkedin.openhouse.common.stats.model;
+
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.NonNull;
+
+/**
+ * Data model for openhousedatasetcommiteventpartitions table.
+ *
+ * <p>Stores partition-level information for each commit. Maps commit events to specific partitions
+ * affected by that commit. One commit can correspond to multiple partition records.
+ *
+ * <p><b>Naming</b>: Represents "partitions of a commit event". This aligns with the conceptual
+ * model where CommitEvent has a "partitions" field that's been normalized into a separate table.
+ *
+ * <p><b>Cardinality</b>: N partition records linked to 1 commit event via commitId foreign key.
+ *
+ * @see CommitEvent
+ */
+@Data
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+public class CommitEventPartitions {
```

Comment:
Should this extend BaseCommitEvent ? I don't understand why we only have commitId here. Is it optimization, if yes, I would suggest from query simplicity let's denormalize and extend BaseCommitEvent